tools/cfn2ts/lib/genspec.ts

```diff
@@ -189,38 +190,11 @@ export function attributeDefinition(resourceName: CodeName, attributeName: strin
 /**
  * Return an attribute definition name for the RefKind for this class
  */
-export function refAttributeDefinition(resourceName: CodeName, refKind: string): Attribute {
-  const propertyName = codemaker.toCamelCase(descriptiveAttributeName(resourceName, refKind));
+export function refAttributeDefinition(): Attribute {
```

Comment:
feels like this can be omitted and just hard-coded in the codegen

Reply:
I think we can just delete this and let `this.ref` work from CfnElement, if we are going with the name change to remove `attr` prefix?

Reply:
Yes of course. Totally redundant now